engine/time/src/main/java/io/deephaven/time/calendar/Calendar.java

```diff
@@ -117,6 +119,7 @@ public DayOfWeek dayOfWeek(final LocalDate date) {
         return DateTimeUtils.dayOfWeek(date);
     }
 
+    // TODO: should this return a String?
```

Comment:
All of the overloads for this method return a `DayOfWeek`. The one that takes string as input doesn't have to be special. It should conform to the behavior the overloads have.